views/index.ejs

```diff
@@ -47,65 +45,233 @@
-              displayResonse(r);
-              console.log('done', r);
+          return webln.sendPayment(payment_request)
+            .then((r) => {
+              postConsole(r)
+            })
+            .then(() => {
+              let invoices = JSON.parse(window.sessionStorage.getItem('invoices'))
+              invoices = invoices.filter(invoice => invoice.id !== id);
+              showInvoices(invoices);
+              window.sessionStorage.setItem('invoices', JSON.stringify(invoices));
             })
-            .catch(function (e) {
-              displayResonse(e);
+            .catch((e) => {
+              postConsole(e)
               alert("Failed: " + e.message);
               console.log('err pay:', e);
             });
         })
-        .catch(function (e) {
+        .catch((e) => {
           alert("Webln error, check console");
+          postConsole(e)
           console.log('err, provider', e);
         });
     }
 
-    window.addEventListener('DOMContentLoaded', function (event) {
-      window.setTimeout(function () {
+    function showInvoices(invoices) {
+      let invoiceList = document.createElement("UL");
+      invoiceList.classList.add('invoice-list');
+      invoices.forEach(invoice => {
+        let listItem = document.createElement("LI");
+        listItem.classList.add('invoice-list-item');
+        listItem.innerHTML = `
+        <span>
+          Name: ${invoice.name}
+        </span>
+        <span>
+          Amount: ${invoice.amount}
+        </span>
+        `
+        listItem.addEventListener('click', () => pay(invoice.id, invoice.data.payment_request))
+        invoiceList.appendChild(listItem);
+      });
+      document.getElementsByClassName('payment-modal-body')[0].innerHTML = '';
+      document.getElementsByClassName('payment-modal-body')[0].appendChild(invoiceList);
+    }
+
+    function enable() {
+      WebLN.requestProvider()
+        .then(function (webln) {
+          webln.getInfo().then(function (info) {
+            postConsole(info)
+          });
+        })
+        .catch(function (e) {
+          postConsole(e)
+          alert("Failed: " + e.message);
+          console.log('error enable:', e);
+        });
+    }
+
+    window.addEventListener('DOMContentLoaded', (event) => {
+      let invoices, consoleResponses;
+      if (window.sessionStorage.getItem('invoices') && window.sessionStorage.getItem('consoleResponses')) {
+        invoices = JSON.parse(window.sessionStorage.getItem('invoices'));
+        consoleResponses = JSON.parse(window.sessionStorage.getItem('consoleResponses'));
+      } else {
+        invoices = [];
+        consoleResponses = [];
+        window.sessionStorage.setItem('invoices', JSON.stringify(invoices));
+        window.sessionStorage.setItem('consoleResponses', JSON.stringify(consoleResponses));
+      }
+
+      let newInvoice = '<%- JSON.stringify(newInvoice) %>';
+      newInvoice = JSON.parse(newInvoice);
```

Comment:
I think something like this should be good enough:
`let newInvoice = <%- JSON.stringify(newInvoice) %>;`